### DISCUSSION TITLE
Why is there no getFirst() on QueryEntity?

### DISCUSSION THREAD

**Discussion by PopePAF** [salesforce/akita#651] [Q&A]
Am I missing something here? 
There is a method selectFirst() for observing the first entity in store.
But it seems like there is no such thing for getting the plain entity without being observed...

**Comment by NetanelBasal** [comment]
To make the library small as possible, we don't expose a getter function for each one of the observation methods. We make this decision because these are uncommon use cases. 

You can do easily do it by calling: `this.query.getAll()[0]`

**Reply by PopePAF** [reply]
Is there no performance loss if my store has a lot of entities? Or would a getFirst() method do the same internally?
(Anyways I would use it in a case where I always have just a single entity in the store... for some kind of a general config I get from the server... there might be a better solution to handle that... so that might be the issue in the first place... )

**Reply by PopePAF** [reply]
I assume I could set it to be the activeEntity and then use the getter for that... just to have a cleaner syntax.
